README.md

```diff
@@ -36,6 +36,94 @@ See the Makefile for instructions.
 For further installation simply copy the resulting binary and man page
 (in the 'doc' directory) to appropriate directories.
 
+## Ringserver Container Image
+
+If you prefer to work with ringserver as a container, that is fully supported.
+Recent releases have containers available publicy as:
+
+[public.ecr.aws/earthscope/ringserver](https://gallery.ecr.aws/earthscope/ringserver)
+
+To pull this image with Docker and tag it locally as "ringserver" for convenience:
+
+```
+docker pull public.ecr.aws/earthscope/ringserver:latest
+docker tag public.ecr.aws/earthscope/ringserver:latest ringserver:latest
+```
+
+To run the default configuration of ringserver with Docker which listens locally on ports 16000 and 18000:
+
+```
+docker run -p 16000:16000 -p 18000:18000 ringserver
+```
+
```

Comment:
Consider adding another section describing how to test the containerized ringserver after the container has been started. 

Use the `slinktool` program to verify that the Ringserver is running. 

`slinktool -Q -v localhost:host`

The command should return:

```
slinktool version: 4.3
[localhost:18000] network socket opened
[localhost:18000] connected to: SeedLink v3.1 (2020.075 RingServer) 
[localhost:18000] capabilities: SLPROTO:3.1 CAP EXTREPLY NSWILDCARD BATCH WS:13
[localhost:18000] organization: Ring Server
[localhost:18000] requesting INFO level STREAMS
2024.212.09:39:40.0, seq 0, Received Info (terminated) blockette
[localhost:18000] network socket closed
```